tests/autoyast/login.pm

```diff
@@ -14,12 +14,12 @@
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
 # Summary: Log into system installed with autoyast
-# Maintainer: Pavel Sladek <psladek@suse.cz>
+# Maintainer: Oliver Kurz <okurz@suse.de>
```

Comment:
see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6642/commits/66fef8a2024cc916a79602f387fda1029c9716a7 for reasoning